x-pack/legacy/plugins/siem/public/mock/kibana_config.ts

```diff
@@ -29,6 +30,61 @@ export const getMockKibanaUiSetting = (config: MockFrameworks) => (key: string)
     return ['8.0.0'];
   } else if (key === DEFAULT_TIMEZONE_BROWSER) {
     return config && config.timezone ? [config.timezone] : ['America/New_York'];
+  } else if (key === DEFAULT_TIMEPICKER_QUICK_RANGES) {
+    return [
+      [
+        {
+          from: 'now/d',
+          to: 'now/d',
+          display: 'Today',
+        },
+        {
+          from: 'now/w',
+          to: 'now/w',
+          display: 'This week',
+        },
+        {
+          from: 'now-15m',
+          to: 'now',
+          display: 'Last 15 minutes',
+        },
+        {
+          from: 'now-30m',
+          to: 'now',
+          display: 'Last 30 minutes',
+        },
+        {
+          from: 'now-1h',
+          to: 'now',
+          display: 'Last 1 hour',
+        },
+        {
+          from: 'now-24h',
+          to: 'now',
+          display: 'Last 24 hours',
+        },
+        {
+          from: 'now-7d',
+          to: 'now',
+          display: 'Last 7 days',
+        },
+        {
+          from: 'now-30d',
+          to: 'now',
+          display: 'Last 30 days',
+        },
+        {
+          from: 'now-90d',
+          to: 'now',
+          display: 'Last 90 days',
+        },
+        {
+          from: 'now-1y',
+          to: 'now',
+          display: 'Last 1 year',
+        },
+      ],
+    ];
```

Comment:
Might just be able to import these `quickRanges` default from:

https://github.com/elastic/kibana/blob/6383a4b693491ddfa8a28ed904b5347e825ebc71/src/legacy/core_plugins/kibana/ui_setting_defaults.js#L860-L914

Reply:
I tried it, I was not able to get the import working